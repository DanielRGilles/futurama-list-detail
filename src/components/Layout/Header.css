.header {
    display: grid;
    grid-template-columns: 1fr 1fr 1fr;
    justify-content: center;
    align-content: center;
    box-sizing: border-box;
    background-color: crimson;
    padding: 5px;
    border: 2px dashed black;
    min-height: 100px;
    
}
.links {
    font-size: larger;
    padding: 15px;
    margin-bottom: 15px;
    line-height: 90%;
}