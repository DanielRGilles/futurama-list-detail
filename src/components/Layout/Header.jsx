import React from 'react'
import { Link } from 'react-router-dom'
import styles from './Header.css'

export default function Header() {
    return (
        <div className={styles.header}>
          <div className={styles.links}> <div><Link to='/'>Home</Link> </div>
          <div><Link to='/characters'>Characters</Link> </div></div>
          <div>
          <h1>Pokemon Pokedex</h1> 
          <section >
         
          </section>
          </div>
          <div></div>
        </div>
    )
}
