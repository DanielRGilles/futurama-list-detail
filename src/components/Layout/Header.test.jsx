import {render } from  '@testing-library/react';
import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min';
import Header from './Header';

describe('Header component test', () => {
  
    it('renders the top nav header', () => {
       const { container } = render(
            <MemoryRouter>
                
                     <Header/>
               
            </MemoryRouter>
        );
        expect(container).toMatchSnapshot();
         
    })

})