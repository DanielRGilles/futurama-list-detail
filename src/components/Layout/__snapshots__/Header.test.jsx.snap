// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test renders the top nav header 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="links"
    >
       
      <div>
        <a
          href="/"
        >
          Home
        </a>
         
      </div>
      <div>
        <a
          href="/characters"
        >
          Characters
        </a>
         
      </div>
    </div>
    <div>
      <h1>
        Pokemon Pokedex
      </h1>
      <section />
    </div>
    <div />
  </div>
</div>
`;
